templates/main.html

```diff
@@ -60,7 +60,7 @@ <h1 class="text-center display-2">
             </div>
             <div class="row">
                 {% if name %}
-                <kbd> {{ name }} </kbd>
+                <h3 class="display-4"> {{ name }} | {{ dept }} </h3>
```

Comment:
Why this? :(

Reply:
Hey! I saw that it was added in a recent commit by you. I wonder why? `kbd` is very small and makes the text smaller than even the labels on top of the timetable. It also renders differently on various browsers. Wouldn't it better to use bigger font than the timetable labels and make it the largest text on the result page so that a user can quickly see who's timetable it is? (especially when they come to the page through the URL parameter flow)